assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/styles.scss

```diff
@@ -8,6 +8,41 @@
 			padding: 1rem;
 			height: 100px;
 		}
+		tr {
+			background: var(--ee-color-grey-14);
+			&:hover {
+				background: var(--ee-color-grey-13);
+			}
+			&:nth-child(even) {
+				background: var(--ee-color-grey-13);
+				&:hover {
+					background: var(--ee-color-grey-12);
+				}
+			}
+		}
+		td {
+			&.date-cell {
+				border-right: 5px solid var(--ee-color-white);
+				padding: var(--ee-padding-small);
+				width: 300px;
+				position: relative;
+				.date-stripe {
+					display: inline-block;
+					height: 100%;
+					width: 5px;
+					position: absolute;
+					top: 0;
+					left: -5px;
+				}
```

Comment:
5px won't be wide enough for the status stripe

```suggestion
				.date-stripe {
					display: inline-block;
					height: 100%;
					width: var(--ee-padding-small);
					position: absolute;
					top: 0;
					left: 0;
				}
```

if need be you can increase the left padding for the parent date-cell like: 

```css
padding: var(--ee-padding-small) var(--ee-padding-small) var(--ee-padding-small) var(--ee-padding-default)
```

Reply:
applied that padding:

![image](https://user-images.githubusercontent.com/1477453/78671288-c2406500-78e7-11ea-9518-2990821b166e.png)